lisa/util/shell.py

```diff
@@ -57,6 +58,40 @@ def wait_tcp_port_ready(
     return is_ready, result
 
 
+def wait_for_ssh_to_be_ready(
+    address: str,
+    port: int,
+    log: Optional[Logger] = None,
+    timeout: int = 300,
+    retry_interval: int = 10,
+) -> bool:
+    is_ready = False
+    client = paramiko.SSHClient()
+    client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
```

Comment:
use paramiko.MissingHostKeyPolicy(). The AutoAddPolicy saves the public key and IP pairs, but it may conflict, because the public IP may be assigned to another server.